### PR TITLE
[Update 8] - Add some breaking changes

### PR DESCRIPTION
**Pull request by MaryamZi**
## Purpose
$title.

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -30,6 +30,64 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
 
 ## Backward-incompatible changes
 
+- A type-checking bug that resulted in incorrect subtype relationships between records with optional fields and open records has been fixed.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:33]:
```suggestion
- Fixed a type-checking bug that resulted in incorrect subtype relationships between records with optional fields and open records.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -30,6 +30,64 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
...
+
+    public function main() {
+        Person person = {name: "May"};
+        Employee employee = person; // Compilation error now.
+
+        io:println(person is Employee); // Prints `false` now.
+    }
+    ```
+
+- Analysis in the `init` method of an `isolated` object has been updated to disallow invalid transferring in/out of values that violated the isolated root invariant.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:55]:
```suggestion
- Updated the analysis in the `init` method of an `isolated` object to disallow invalid transferring in/out of values that violated the isolated root invariant.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 3]
```diff
@@ -30,6 +30,64 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
...
+        function init(Node node) {
+            self.node = node.clone();
+
+            self.arr.push(node); // Compilation error now.
+            self.arr[0] = node; // Compilation error now.
+        }
+    }
+    ```
+
+- Mutable objects have been disallowed in annotations.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:76]:
```suggestion
- Disallow mutable objects in annotations.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md (hunk 1 above)
Comment (MaryamZi) [reply downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:33]: Again, please see https://github.com/ballerina-platform/ballerina-dev-website/pull/7642#discussion_r1305092375. Rest is for consistency.

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md (hunk 2 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:55]: Closing as per https://github.com/ballerina-platform/ballerina-dev-website/pull/7669#discussion_r1305147559.

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md (hunk 3 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:76]: Resolving as per https://github.com/ballerina-platform/ballerina-dev-website/pull/7669#discussion_r1305147559.

**Review by praneesha — APPROVED** [review]

**Review by hasithaa — APPROVED** [review]

**Pull request merged**